src/components/OcBreadcrumb.vue

```diff
@@ -143,12 +143,17 @@ export default {
 
     > :last-child > span {
       color: var(--oc-color-text-default);
+
+      &:hover {
+        text-decoration: none;
+      }
     }
 
     > li a:hover,
     > li span:hover,
     > li button:hover {
-      color: var(--oc-color-swatch-primary-default);
+      color: var(--oc-color-text-default);
+      text-decoration: underline;
```

Comment:
This would also set a hover for a `span` element. Not sure if we want this, what was the idea behind it?

Reply:
This was already implemented, I just changed the color reference to meet a11y requirements. I also added an underline on hover which was not requested, but makes it a bit more obvious it's a clickable link. No problem to remove it again! 🙂

The last `span` element inside a `li` element will have no text decoration or any other hover effect, see line 147 in the same file.